static/js/modules/tables.js

```diff
@@ -589,8 +589,9 @@ DataTable.prototype.fetch = function(data, callback) {
 
 DataTable.prototype.export = function() {
   var url = this.buildUrl(this.api.ajax.params(), false);
-  download.download(url);
+  var item = download.download(url);
   this.disableExport({message: DOWNLOAD_MESSAGES.pending});
+  item.$body.find('.button').focus();
```

Comment:
I think it would be less confusing to (1) expose a method on `DownloadItem` that focuses on the button, or (2) add a parameter to the `download` helper indicating whether the button should be focused after draw. Either approach would hide unimportant details of the `DownloadItem` implementation from devs who only want to work on tables--like the fact that `DownloadItem` has an attribute called `$body`, or that the relevant selector to focus on is called `.button`.